### PR TITLE
Add avif image support

### PR DESCRIPTION
**Pull request by vandenbosm**
Avif is a new image format that can reduce image sizes by 50% of even WebP image sizes.  Currently supported by Chrome > v85 and will soon have support in other browsers.   This will help to increase performance and minimize asset sizes.

Here's a great write-up about it:
https://jakearchibald.com/2020/avif-has-landed/

### CODE REVIEW

**Comment by vandenbosm** [comment]
Sorry, was cleaning up my branches and didn't realize it was going to auto-close this PR.

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ node_modules
 # Twelvety caches responsive images
 # Don't commit otherwise your build won't generate any images
 .twelvety.cache
+yarn.lock
```
Comment (gregives) [inline .gitignore:7]: I'm not sure if we want to ignore `yarn.lock`, in case someone wants to switch from npm to Yarn. What do you think?

**Review by gregives — COMMENTED** [review]
Thanks for another great pull request! I've just got one question about `.gitignore`

File: .gitignore (hunk 1 above)
Comment (vandenbosm) [reply .gitignore:7]: You don't have to include it; I couldn't remove it from my commit and assumed you would ignore it during the PR if you didn't want.   From a technical side, you can ignore package.lock and yarn.lock as both are dynamically created on install and don't need to be part of any repository.

**Comment by vandenbosm** [comment]
Thanks for providing a great 11ty template! I'm happy to contribute 🍻

File: .gitignore (hunk 1 above)
Comment (gregives) [reply .gitignore]: That's alright, I've removed it from this pull request but users can add it locally if they want. It's usually best practice to commit lock files https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

**Comment by gregives** [comment]
Thank you very much!

File: .gitignore (hunk 1 above)
Comment (vandenbosm) [reply .gitignore]: Thanks for the article!

**Pull request merged**